.gitignore

```diff
@@ -0,0 +1,104 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
This feels like something that would be in everyone's `~/.gitignore` file. (I mean, we can add it, whatever, but it's just a bit strange)

Reply:
it's the standard gitignore file of github.
It removes most files which shouldn't be in a git repository. So the section names

Reply:
Hmm, I didn't realise that github actually encouraged that. Fair enough